ta/gatekeeper/gatekeeper_ta.c

```diff
@@ -0,0 +1,767 @@
+// SPDX-License-Identifier: Apache-2.0
+/* Copyright (c) 2018, Linaro Limited */
+/* Copyright (c) 2017, GlobalLogic  */
+
+#include <tee_internal_api.h>
+#include <tee_internal_api_extensions.h>
+#include <utee_defines.h>
+
+#include "failure_record.h"
+#include "gatekeeper_ipc.h"
+#include "ta_gatekeeper.h"
+
+static const uint8_t secret_id[] = {0xB1, 0x6B, 0x00, 0xB5};
+
+TEE_Result TA_CreateEntryPoint(void)
+{
+	TEE_Result res = TEE_SUCCESS;
+	TEE_ObjectHandle secret_obj = TEE_HANDLE_NULL;
+	uint8_t secret_data[TEE_SHA256_HASH_SIZE];
+
+	DMSG("Checking master key secret");
+	res = TEE_OpenPersistentObject(TEE_STORAGE_PRIVATE, secret_id,
+				       sizeof(secret_id),
+				       TEE_DATA_FLAG_ACCESS_READ,
+				       &secret_obj);
+
+	if (!res) {
+		DMSG("Secret is already created");
+		goto exit;
+	}
+
+	if (res != TEE_ERROR_ITEM_NOT_FOUND) {
+		EMSG("Failed to open secret, error=%X", res);
+		return res;
+	}
+
+	DMSG("Create master key secret");
```

Comment:
I would remove this and just keep the EMSG at 48. I.e., if all goes well, then it's is quiet, if not an error is shown.

Reply:
fixed